L1Trigger/L1TMuonEndCap/src/PatternRecognition.cc

```diff
@@ -287,6 +287,8 @@ bool PatternRecognition::is_zone_empty(
     EMTFHitCollection::const_iterator conv_hits_end = ext_conv_hits_it->end();
 
     for (; conv_hits_it != conv_hits_end; ++conv_hits_it) {
```

Comment:
I do not know if it brings something in performance as well. Otherwise style and the +1 of David.